std/http/cookie.ts

```diff
@@ -92,6 +93,26 @@ function validateCookieName(name: string | undefined | null): void {
   }
 }
 
+/**
+ * Validate Path Value.
+ * @param path Path value.
+ */
+function validatePath(path: string | null): void {
+  if (path == null) {
+    return;
+  }
+  for (let i = 0; i < path.length; i++) {
+    const c = path.charAt(i);
+    if (
+      c < String.fromCharCode(0x20) || c > String.fromCharCode(0x7E) || c == ";"
```

Comment:
Reference?

Reply:
https://tools.ietf.org/html/rfc6265#section-4.1.2.4

Reply:
Okay, found it in https://tools.ietf.org/html/rfc6265#section-4.1.1:
```
path-value        = <any CHAR except CTLs or ";">
```
(where a `CHAR` is an ASCII character).

Reply:
Please add this reference as a comment